markhor_description/launch/display_markhor_standalone.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
```

Comment:
This file and display_markhor.launch have unintuitive names to me. I'm not sure what scenarios they each cover, so I can't suggest anything.

Reply:
display_markhor opens rviz with markhor in it,  display_markhor_standalone does that + publishes joint states and robot state using sliders in the ui, this is mostly used to verify URDF functionality

Reply:
In that case, going from display_markhor_standalone.launch to display_markhor_with_jointstates.launch would have a similar length and state its use outright. What do you think?